### DISCUSSION TITLE
A: plugin/auto: no next plugin found

### DISCUSSION THREAD

**Discussion by prologic** [coredns/coredns#5297] [Q&A] [answered]
HI all 👋

Benn pulling my teeth over this since yesterday, and I can't for the life of me figure out what's wrong with this setup. It seems simple enough. Set Up a couple of authoritative zones, serve them. No caching or forwarding necessary as I have other DNS infra that will forward the requests to CoreDNS.

What is wrong with this config that ends in the error:

```
A: plugin/auto: no next plugin found
```

`Corefile`:
```
.:53 {
    auto home.arap {
        directory /data/zones
    }
    health
    prometheus 0.0.0.0:9153
    errors stdout
    log stdout
}
```

`db.home.arpa`:
```
$TTL 60
$ORIGIN home.arpa.
@           IN  SOA  home.arpa. admin.home.arpa. (
                     2022040200 ; Serial
                     4H         ; Refresh
                     1H         ; Retry
                     7D         ; Expire
                     4H )       ; Negative Cache TTL


@                         IN  A    127.0.0.1
*.home.arpa.              IN  A    127.0.0.1
salty.home.arpa.          IN  A    127.0.0.1
_salty._tcp.home.arpa.    IN  SRV  0 0 443 salty.home.arpa.
_avatars._tcp.home.arpa.  IN  SRV  0 0 443 salty.home.arpa.
```

Hare are the logs from coredns:

```
[INFO] plugin/auto: Inserting zone `home.arpa.' from: /data/zones/db.home.arpa
.:53
CoreDNS-1.9.1
linux/amd64, go1.17.8, 4b597f8
[ERROR] plugin/errors: 2 home.arpa. A: plugin/auto: no next plugin found
```

**Comment by chrisohaver** [comment, marked as answer]
arpa/arap typo? (On auto plugin line)

**Reply by chrisohaver** [reply]
A lesser issue, the first parameter for log specifies the zone that will be logged, not the output location. So log stdout will only log queries ending in .stdout.

>

**Reply by prologic** [reply]
Oh thanks!

**Comment by prologic** [comment]
Turns out I'm just a dumb fucking blind idiot (okay the blind part is true 😅) geez lious. I can't believe I typo'd the stupid Zone!
